voicebox/connection.py

```diff
@@ -69,10 +92,18 @@ def receive_data(self):
             except ConnectionResetError:
                 logging.error("Connection to client lost: Adviced to kill connection")
             except OSError:
-
                 logging.info("Channel already closed: Adviced to kill connection")
 
     def send_message(self, message, msg_type=1):
```

Comment:
The `send_message` method in the `Connection` class does not return any value. It would be beneficial to return a status message or a boolean value indicating the success or failure of the operation. [medium]